checks/collector.py

```diff
@@ -529,6 +529,8 @@ def run(self, checksd=None, start_event=True, configs_reloaded=False):
             del data['metrics']
             del data['events']
             del data['service_checks']
+            if data.get('processes'):
+                data['processes']['apiKey'] = '*************************' + data['processes'].get('apiKey', '')[-5:]
```

Comment:
Minor nit: maybe do `'*' * 27` to make the scrubbed key the proper length (32)

Reply:
Don't worry since the flare also doesn't output the proper length https://github.com/DataDog/dd-agent/blob/1d0dc34ce7dd3f114ddeed99540b655d97ee879b/utils/flare.py#L105